xbmc/interfaces/python/XBPython.cpp

```diff
@@ -581,6 +581,10 @@ bool XBPython::OnScriptInitialized(ILanguageInvoker *invoker)
       CLog::Log(LOGDEBUG, "PYTHONHOME -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
       CLog::Log(LOGDEBUG, "PYTHONPATH -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
     }
+
+#if defined(TARGET_DARWIN)
```

Comment:
the fix for Android here was wrong and should go away. SSL_CERT_FILE is not a python thing, hence this is a bad spot to add it. Environment settings should all be in a single place. That is where app is started or before.